.eslintrc

```diff
@@ -0,0 +1,10 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "sourceType": "module"
```

Comment:
@amaxama I made sure this config is as simple as possible—everything in here is required! The `sourceType` and `es2021` parts are so we can use `import` statements